src/containers/OrderBook/index.tsx

```diff
@@ -208,89 +124,116 @@ class OrderBookContainer extends React.Component<Props, State> {
 
                     if (isMobileDevice) {
                         return [
-                            <span key={i}><Decimal fixed={priceFixed} prevValue={array[i + 1] ? array[i + 1][0] : 0}>{price}</Decimal></span>,
-                            <Decimal key={i} fixed={amountFixed}>{total[i]}</Decimal>,
+                            <OrderBookTableRow
+                                type="price"
+                                prevValue={array[i + 1] ? array[i + 1][0] : 0}
+                                price={price}
+                                fixed={priceFixed}
+                            />,
+                              <OrderBookTableRow
+                                total={total[i]}
+                                fixed={amountFixed}
+                              />,
```

Comment:
fix indentation

Reply:
I think linter should fix it, in case it allows to commit and push it means everything is fine with an indentation